receiver/elasticapmintakereceiver/internal/mappers/intakeV2ToSemConv.go

```diff
@@ -97,6 +101,123 @@ func TranslateIntakeV2SpanToOTelAttributes(event *modelpb.APMEvent, attributes p
 	}
 }
 
+func translateCloudAttributes(event *modelpb.APMEvent, attributes pcommon.Map) {
+	if event.Cloud != nil {
+		if event.Cloud.Provider != "" {
+			attributes.PutStr(semconv.AttributeCloudProvider, event.Cloud.Provider)
+		}
+		if event.Cloud.Region != "" {
+			attributes.PutStr(semconv.AttributeCloudRegion, event.Cloud.Region)
+		}
+		if event.Cloud.AvailabilityZone != "" {
+			attributes.PutStr(semconv.AttributeCloudAvailabilityZone, event.Cloud.AvailabilityZone)
+		}
+		if event.Cloud.AccountId != "" {
+			attributes.PutStr(semconv.AttributeCloudAccountID, event.Cloud.AccountId)
+		}
+		if event.Cloud.ServiceName != "" {
+			attributes.PutStr(semconv.AttributeCloudPlatform, event.Cloud.ServiceName)
```

Comment:
This won't be 100% SemConv compatible, but this is fine.

[SemConv spec](https://opentelemetry.io/docs/specs/semconv/registry/attributes/cloud/#cloud-platform) says: 
> cloud.platform: The prefix of the service SHOULD match the one specified in cloud.provider.

while [`Service.Name`](https://www.elastic.co/docs/reference/ecs/ecs-cloud#field-cloud-service-name) does not have the prefix.

Just noting this here, since I looked this up. Overall, I think this is fine - it's a `SHOULD` in the spec, so I'd say it's acceptable.